composer.json

```diff
@@ -21,15 +21,19 @@
         "drupal/console": "^1.0"
     },
     "require": {
-        "composer/installers": "^1.0",
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "cweagans/composer-patches": "^1.6.0",
-        "acquia/lightning": "~2.1.0"
+        "acquia/lightning": "~2.1.0",
+        "oomphinc/composer-installers-extender": "^1.1"
```

Comment:
We were directly requiring composer/installers before (which this supersedes), so as far as I can tell, it's a root requirement...we could open a follow-up PR to experiment with removing it, though.